Tests/SwiftGRPCTests/GRPCTests.swift

```diff
@@ -157,23 +166,32 @@ func callUnary(channel: Channel) throws {
     try call.start(.unary, metadata: metadata, message: message) {
       response in
       // verify the basic response from the server
-      XCTAssertEqual(response.statusCode, (i % 2  == 0) ? evenStatusCode : oddStatusCode)
```

Comment:
This means we never check for non `.ok` status codes which was the original bug filed in #144.